src/components/DisplayHeader.jsx

```diff
@@ -23,9 +24,10 @@ const DisplayHeader = props => {
         >
           {name}
         </Menu.Item>
-        <Menu.Item name="main-header" id="main-header" position="right">
+        <Image src={Logo} size='medium' centered rounded/>
+        {/* <Menu.Item name="main-header" id="main-header" position="right">
           <Icon name="sidebar" size="large"></Icon>
```

Comment:
Do we need this commented out code?

Reply:
We have not yet used the functionality of the hamburger menu but were thinking we could implement it when we finalize the styling for the final iteration.

Reply:
We have decided to remove it and add it as a chore. Thanks!